.github/workflows/test_python-client.yml

```diff
@@ -0,0 +1,50 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: Test - python client
+on:
+  pull_request:
+    branches:
+        - master
+        - 'v[0-9]+.*' # release branch
+        - ci-test # testing branch for github action
+        - '*dev'      # developing branch
+    paths:
+      - python-client/**
+
+  workflow_dispatch:
```

Comment:
is it useable?

Reply:
Yes, it is used for manually triggering workflow. For example, you can click the "Run workflow" button on site https://github.com/apache/incubator-pegasus/actions/workflows/lint_and_test_admin-cli_always_pass.yml

Reply:
great!